docs/assets/css/slide.css

```diff
@@ -0,0 +1,99 @@
+
+
+@-webkit-keyframes scroll {
+    0% {
+      -webkit-transform: translateX(0);
+              transform: translateX(0);
+    }
+    100% {
+      -webkit-transform: translateX(calc(-250px * 5));

```

Comment:
This smells like CSS animations to me.... now, we're talking, @Neilblaze. 😄

Reply:
Yup it is! :smile: 
There were some 3rd party .js scripts that came into my mind like [Slick.js](https://kenwheeler.github.io/slick/), but yeah, why add 300 lines of extra Javascript when we can make it in less than 50 lines of CSS :laughing: